.babelrc

```diff
@@ -1,4 +1,8 @@
 {
-  "plugins": ["./data/schema"],
-  "presets": ["react-native"]
-}
+  "plugins": [
+    "./data/schema"
+  ],
+  "presets": [
+    "react-native"
+  ]
+}
```

Comment:
I think you don't have the pre-commit hooks set up, run these: https://github.com/artsy/emission/blob/master/Makefile#L34-L36